### DISCUSSION TITLE
Is APE the new Brownie?

### DISCUSSION THREAD

**Discussion by abstractumx** [ApeWorX/ape#695] [General]
Hello everyone, 

looking around some github repos I've found out this "Smart contract framework compiler" using Python.

Will he be Brownie's promising new successor?

**Comment by fubuloubu** [comment]
Hello @abstractumx! We will be making an announcement soon from Brownie's official channels, but yes indeed the plan here is to pass the torch to Ape. You can rely on us to continue building out the Python-based dev framework for all your smart contract needs
